docs/server/source/cloud-deployment-templates/server-tls-certificate.rst

```diff
@@ -1,86 +1,28 @@
-Generate Cluster Member Certificate for MongoDB
-===============================================
+How to Generate a Server Certificate for MongoDB
+================================================
 
```

Comment:
:+1: The opening paragraph of that page explains how the MongoDB documentation calls server certificates "member certificates" so readers will understand.